### DISCUSSION TITLE
Sound outer marker

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4452] [Ideas]
Hello developers,

is it planned to eliminate the outer marker sound? Sorry for writing here in the discussion area ... I am really short of time to fill out the forms in the issue and pull request area ...

Greets klaus

**Comment by hotshotp** [comment]
we cannot do it because it is in the default game sound files and if we turned it off there it would affect the entire sim. so apologies but it has to stay :(
